src/components/donation/Steps/DonateMethods/Stripe/Incrementers.tsx

```diff
@@ -0,0 +1,31 @@
+import { useFormContext } from "react-hook-form";
+import { FormValues } from "./types";
+
+export default function Incrementers() {
+  return (
+    <div className="flex justify-center gap-3">
+      <Incrementer value={40} />
+      <Incrementer value={100} />
+      <Incrementer value={200} />
+    </div>
+  );
+}
+
+function Incrementer({ value }: { value: number }) {
+  const { setValue, trigger, watch } = useFormContext<FormValues>();
```

Comment:
```suggestion
  const { setValue, trigger, getValues } = useFormContext<FormValues>();
```
for one time (on user click) retrieval, it's correct (in context of `react-hook-form` - not necessarily better ) to use [`getValues("path")`](https://react-hook-form.com/docs/useform/getvalues)

Reply:
This is just what I needed to:
- read the value, but
- not validate the whole form

Thanks!